model/client4_test.go

```diff
@@ -12,6 +12,12 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+// https://github.com/mattermost/mattermost-plugin-starter-template/issues/115
+func TestClient4TrimTrailingSlash(t *testing.T) {
+	client := NewAPIv4Client("https://foo.com:1234/")
+	assert.False(t, strings.HasSuffix(client.Url, "/"))
```

Comment:
Can we be more explicit here, and test the expected URL /and/ the expected API url? The latter, I think, is the most important part of this change:
```suggestion
 	assert.Equal(t, "https://foo.com:1234", client.Url)
 	assert.Equal(t, "https://foo.com:1234/api/v4", client.ApiUrl)
```

Reply:
Certainly.  I also updated the case to stress the three likely inputs: 
- 0 slashes (which is expected), 
- 1 slash (which is probable), and 
- 5 slashes (unlikely but possible if you fell asleep at the wheel). An arbitrary number to demonstrate it's removing "all"